### DISCUSSION TITLE
Ideal size/orientation for cover images

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#457] [Get Help] [answered]
Is there an ideal size or ratio for cover images so that they render properly across devices and formats?

**Comment by geealbers** [comment, marked as answer]
A landscape orientation works best (wider than it is tall), and I like to use something at a minimum of 1800px. _ will push up to 2200px or 2400px, but I will also play with the JPG settings/compression to keep the file size at a minimum while still being crisp enough. And finally, having the focal point more centered works best across screen sizes, without having to do a lot of custom CSS work.
